### PR TITLE
rsync: update page

### PR DESCRIPTION
**Pull request by harryhanYuhao**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Line 4: change `host` to `user@host`
Line 13: change `--archive to -a` 
Delete 19-22: as `-a` option includes `--recursive` option. I think `-a` is used most freqently, and rarely we want recursively clone without archive mode.
Line 29: Delete `--recursive` option as it is redundant.
Line 37: remove extra `-`. flag `--rsh` is invalid.

### CODE REVIEW

**Comment by acuteenvy** [comment]
> change host to user@host

OK

---

> change --archive to -a

We prefer using long options: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax

---

> Delete --recursive option as it is redundant

OK

---

> remove extra -. flag --rsh is invalid.

`--rsh` works on my system. It's also in the man page:
> --rsh=COMMAND, -e        specify the remote shell to use

What `rsync` version are you using?

---

> as -a option includes --recursive option. I think -a is used most freqently, and rarely we want recursively clone without archive mode

Sometimes you don't want to preserve every file attribute, and then your only option is to specify exactly what you want instead of `--archive`. I think this is a more common case than using a jump server.

**Comment by harryhanYuhao** [comment]
> > change host to user@host
> 
> OK
> 
> > change --archive to -a
> 
> We prefer using long options: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax
> 
> > Delete --recursive option as it is redundant
> 
> OK
> 
> > remove extra -. flag --rsh is invalid.
> 
> `--rsh` works on my system. It's also in the man page:
> 
> > --rsh=COMMAND, -e        specify the remote shell to use
> 
> What `rsync` version are you using?
> 
> > as -a option includes --recursive option. I think -a is used most freqently, and rarely we want recursively clone without archive mode
> 
> Sometimes you don't want to preserve every file attribute, and then your only option is to specify exactly what you want instead of `--archive`. I think this is a more common case than using a jump server.

Ok. I have changed back `-a` to `--archive`. 

As to `--rsh`, I confused it for `-rsh`. But I believe it still need a `=`.

I am using rsync version 3.2.7  protocol version 31.

File: pages/common/rsync.md [hunk 1]
```diff
@@ -26,12 +26,12 @@
 
 - Recursively copy directories, use archive mode, resolve symlinks and skip files that are newer on the destination:
```
Comment (acuteenvy) [inline pages/common/rsync.md:27]: Archive mode is explained above.
```suggestion
- Use archive mode, resolve symlinks and skip files that are newer on the destination:
```

File: pages/common/rsync.md [hunk 2]
```diff
@@ -26,12 +26,12 @@
...
+`rsync --archive --update --copy-links {{path/to/source}} {{path/to/destination}}`
 
 - Transfer a directory to a remote host running `rsyncd` and delete files on the destination that do not exist on the source:
 
 `rsync --recursive --delete rsync://{{host}}:{{path/to/source}} {{path/to/destination}}`
 
 - Transfer a file over SSH using a different port than the default (22) and show global progress:
 
-`rsync --rsh 'ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
+`rsync --rsh='ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
```
Comment (acuteenvy) [inline pages/common/rsync.md:37]: I have the same `rsync` version and `--rsh arg` works. Can you try to run it?
```suggestion
`rsync --rsh 'ssh -p {{port}}' --info=progress2 {{host}}:{{path/to/source}} {{path/to/destination}}`
```

Comment (kbdharun) [reply pages/common/rsync.md:37]: Same here, tried it on Ubuntu LTS 22.04, Fedora Workstation 38.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**